.pylintrc

```diff
@@ -0,0 +1,37 @@
+[MESSAGES CONTROL]
+
+disable=broad-except,
+        too-few-public-methods,
+        parse-error,
```

Comment:
This is only needed, because `script.test` is linted.
```
[Errno 2] No such file or directory: 'script.test/__init__.py' (parse-error)
```

Reply:
How can we correct this?

Reply:
Do not lint `script.test`. It is only random Python code, which is used for addon-checker tests.

Reply:
Should we keep this disabled?

Reply:
or should we create __init__.py file?

Reply:
> Do not lint script.test. It is only random Python code, which is used for addon-checker tests.

Neither disable `parse-error` check or create a `__init__.py` file.

Reply:
Okay I did that.